tests/queries/0_stateless/02732_rename_after_processing.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+# Tags: no-parallel
+
```

Comment:
Please, add tests for bad `rename_files_after_processing` setting

Reply:
Fixed, added tests for invalid placeholders